tools/st2ctl

```diff
@@ -122,6 +122,7 @@ function st2start(){
   else
     service mistral start
   fi
+  (cd /opt/stackstorm/static/webui && nohup python -m SimpleHTTPServer 80) &>> ${LOGFILE} &
```

Comment:
@DoriftoShoes I'm pretty sure you have some strong opinions on this part... It looks to me like a best place to put this kind of stuff, though you might have better idea. Also, we need to figure out a way to disable this thing on production so it doesn't reserve 80 port.